README.md

```diff
@@ -21,8 +22,9 @@ Create a Heroku app, set your ngrok token, and push:
 
 ```sh-session
 $ heroku create
+$ heroku buildpacks:add --index 1 heroku-community/apt
```

Comment:
i'd rather not require this, and instead fix whatever problem there was installing screen

Reply:
It occurred during installation.
`screen` could not be installed due to the read-only filesystem I believe.
That led to issues booting up the instance.

It can be replayed too by deploying via the link in the README and checking the server deployment logs. I also did not include any keys for AWS storage.